deploy/docker/production/entrypoint.bash

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+
+# Global Fourfront Application Entrypoint
+# This script resolves which application type is desired based on
+# the "$application_type" environment variable. Possible options are:
+#  * "deployment" to run the deployment
+#  * "indexer" to run the production indexer (forever)
+#  * "portal" to run the production portal worker (API back-end)
+#  * "local" to run a local deployment
+
+# Note that only "local" can be run from the local machine
+# but the same image build is run across the entire local/production stack.
+
+deployment="deployment"
```

Comment:
What's the advantage of doing this over just referencing "deployment" below? In Python it has virtue because you'd get an unbound variable error if you misspelled something. In a shell script you'll just end up comparing to the empty string if you make a typo. I guess it _vaguely_ provides some doc of what states are allowed, but you could just do that in a comment or tell people to read the if statement below.